apps/web/modules/bookings/lib/getBookingInfo.ts

```diff
@@ -0,0 +1,67 @@
+import { getBookingWithResponses } from "@calcom/features/bookings/lib/get-booking";
+import prisma from "@calcom/prisma";
+
+const getBookingInfo = async (uid: string) => {
```

Comment:
Instead of copying and pasting the prisma booking call, we should have the prisma.booking.findFirst as a function that is re-used here and where server side props are generated for web app in "apps/web/modules/bookings/views/bookings-single-view.getServerSideProps.tsx". If we don't we risk having disrepancies.

Reply:
@supalarry created a separate function that handles the prisma call so should be good now :)